### ISSUE TITLE
Unable to Search or Browse Anything

### ISSUE THREAD

**Issue by mangophoenix** [Wilson-Magic/plugin.video.hummingbird#21]
I’m really hoping you can help provide some guidance on how to get this to function. :’D

Fresh install of Kodi V18.6 and Hummingbird 0.5.0. When going into a list that should bring up a list of anime, the wheel spins indefinitely. 

Logged in with Kitsu, MAL, Fanart, and WonderfulSubs. Cleared cache and turned off Auto pick best quality. Tried disabling certain anime providers with no change.  Left spinning for a period of 10 minutes and it did not continue. Windows 10 64-bit device with 16 GB of memory. No errors or other logs seem to be produced. 

Under dependencies, script.module.cryptopy and plugin.video.gdrive show as not available, unsure if that has an effect. 

Please let me know if there is any more information I can give to help or if I’m missing something obvious.

**Comment by Goldenfreddy0703** [comment]
Hey Wilson must very busy for some reason so I'm going to help you. Do you have a good internet connection and do you have any software such as adblockers or antivirus stuff installed, that may affect kodi but I'm not too sure on that but sense you said it was a fresh install, that could be the reason. If you also could, send me a log and I will see what happened.

**Comment by mangophoenix** [comment]
All good. Tried both restarting the computer and completely disabling antivirus. Log is attached.
[kodi.log](https://github.com/Wilson-Magic/plugin.video.hummingbird/files/4524724/kodi.log)

**Comment by Goldenfreddy0703** [comment]
Hey I'm on my phone at the moment but I do believe that the addon is in maintenance at the moment, the dev is busy and it's been like 2 or 3 weeks sense he responded so he is probably working on something big. I believe the addon is on maintenance at the moment becouse several people have been getting playback errors and I do to but use WSbeta or Watchnixtoons until the addon is fixed.

**Comment by Wilson-Magic** [comment]
Hi, sorry for the late reply. 

Looks like something is erroring out at fanart tv. Can you try disabling it and trying again?

**Comment by mangophoenix** [comment]
All good my man. Life is crazy.

That fixed it. I can go in and start anime properly and it does play video. Unsure why that happened. Let me know if there is anything else I can do to help figure out why it does that

**Comment by Wilson-Magic** [comment]
Yeah it's simply a limitation, for some reason my try except doesn't always work. so it just errors out rather than moving on.
